crates/bevy_render/src/pipeline/vertex_buffer_descriptor.rs

```diff
@@ -46,24 +45,8 @@ pub struct VertexAttributeDescriptor {
     pub shader_location: u32,
 }
 
-#[derive(Default)]
-pub struct VertexBufferDescriptors {
-    pub descriptors: HashMap<String, VertexBufferDescriptor>,
-}
-
-impl VertexBufferDescriptors {
-    pub fn set(&mut self, vertex_buffer_descriptor: VertexBufferDescriptor) {
-        self.descriptors.insert(
-            vertex_buffer_descriptor.name.to_string(),
-            vertex_buffer_descriptor,
-        );
-    }
-
-    pub fn get(&self, name: &str) -> Option<&VertexBufferDescriptor> {
-        self.descriptors.get(name)
-    }
-}
-
-pub trait AsVertexBufferDescriptor {
-    fn as_vertex_buffer_descriptor() -> &'static VertexBufferDescriptor;
+pub fn get_vertex_attribute_name_id(name: &str) -> usize {
+    let mut hasher = fnv::FnvHasher::default();
```

Comment:
Rather than pull in another dependency, can we use bevy_utils::AHasher here?

Reply:
Never heard of AHash, but from what I see it's the better fit here